builds/zika/Snakefile

```diff
@@ -1,24 +1,40 @@
+# config variables to be used by rules below
```

Comment:
We should include an `all` rule at the top of each Snakefile that includes one or more final outputs to be built. This special rule allows users to simply type `snakemake` from the proper build directory without needing to specify the output filename (e.g., `zika_tree.json`).